std/process.d

```diff
@@ -3233,11 +3242,15 @@ private:
     assertThrown(environment["std_process"]);
 
     // get() without default value
-    assert (environment.get("std_process") == null);
+    assert (environment.get("std_process") is null);
 
     // get() with default value
     assert (environment.get("std_process", "baz") == "baz");
 
+    // get() on an empty (but present) value
+    environment["std_process"] = "";
+    assert(environment.get("std_process") !is null);
```

Comment:
because assert(null == "") is true, and as such, this would incorrectly pass the current result (null).